src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supply;
+    private int buy;
```

Comment:
let's make these variables local
as I wrote earlier, we don't want the utility classes (that are designed to manipulate the input data) to store any state (class non-constant fields are considered state)